content/downloads/windows/index.md

```diff
@@ -11,39 +11,32 @@ Microsoft Defender SmartScreen, see the [section below](#ms-ss).*
 
 ### Development snapshot builds
 
-You can always check what's cooking on the master branch :)
+You can always see what's cooking on the master branch :)
 
-These builds might be slow or unstable, they are designed with developers
+These builds might be slow or unstable as they are designed with developers
 and testers in mind: [download instructions for snapshot builds](
-https://github.com/dosbox-staging/dosbox-staging#development-snapshot-builds)
+https://github.com/dosbox-staging/dosbox-staging#development-snapshot-builds).
 
 ### Older builds
 
 There are no stable builds yet.
 
 ### <a name="ms-ss"></a> Microsoft Defender SmartScreen
 
-Starting with Windows 8, Microsoft enabled "SmartScreen" technology, which
-disrupts user ability to run new (as in: newly developed) software.
+In 2012, Microsoft added "SmartScreen" to Windows 8, a rent-seeking system
+meant extort payments from developers in exchange for sharing their software
+with Window users via the App Store (see [[1]] and [[2]] below).
 
-Bypassing this check from software distribution side requires paying hundreds
-of dollars for "[Extended Validation Code Signing Certificate][1]" and putting
-the software on Windows Store (code signing only is not enough<sup>[2]</sup>).
+In lieu of paying Microsoft, the dosbox-staging project uses GitHub's free
+integration service to generate Windows release packages. We will also rely on
+Microsoft's "Application Reputation" scheme to eventually whitelist
+dosbox-staging for users (see: [[3]], and the last paragraph in [[4]]).
```

Comment:
Similar to the disclaimer I put in macOS download page, but in here it's even more murky.

Again, I agree that this is basically an extortion scheme - big rake for pushing developers towards Windows Store.

But the way Microsoft designed it - their motive can be plausibly denied at every turn.

They don't profit (directly) from certification - certificates need to be bought from "whitelisted" vendors (Symantec and Broadcom right now? not sure) - and they are *expensive*. I guess MS prefers recurring tax on Windows Store apps instead of flat fee for a certificate - but free apps are allowed on the Store as well (AFAIK).

Finally, everything can be denied by "Application Reputation" system - basically, if users use dosbox-staging it will be whitelisted, but MS is still a gatekeeper.

We will depend *solely* on Application Reputation system to eventually pass the filter. And I guess it'll be back for every new stable version… :(

Anyway - I wanted to not be antagonistic towards users, just communicate: "Hey, we know you probably never seen this screen before, it's not our fault, ok? Just click the button."

Maybe we should drop these sections completely for now and just post the screenshots?

Reply:
Your description here helps a lot - I think we definitely should describe it (and provide screenshots).  Will take a crack at adjusting the wording..

Reply:
PS - I just saw this: 
https://blog.aluxian.com/free-code-signing-certificate-for-open-source-software-d836270823a7, Great news, and in Poland no less! (but then saw they're switch from free to $14/yr.. maybe they'll make an exception for you as a fellow citizen :-)

Reply:
Wording updated!

Reply:
I looked it up some time ago… it's not an option, at least not for me right now. Their certificate for open source projects costs €25, and it does NOT remove SmartScreen - it will only remove "Unknown publisher" from [this screen](https://dosbox-staging.github.io/downloads/windows/smartscreen2.png). The certificates, that bypasses SmartScreen completely costs… (drums) €359… a year.

I should probably set Patreon or something to cover such costs…

Reply:
ugh.. that's brutal.  I can see people eventually donating, but I sure wouldn't want their dollar simply be transfered into the hands of Symantec or Apple. The FSF should buy a single one of these, and then offer signing for all  GPL projects  :-)